tests/examples_tests/conftest.py

```diff
@@ -16,13 +16,15 @@ def findfiles(path, regex):
 
 
 TEST_SUITES = {
-    "chemistry_freezing_isotopes": [
+    "chemistry_freezing_isotopes_a": [
         "Jaruga_and_Pawlowska_2018",
         "Kreidenweis_et_al_2003",
         "Alpert_and_Knopf_2016",
         "Ervens_and_Feingold_2012",
         "Niedermeier_et_al_2014",
         "Bolot_et_al_2013",
+    ],
+    "chemistry_freezing_isotopes_b": [
```

Comment:
the contents are only isotopes, so perhaps let's rename to isotopes?

Reply:
I will move Abade and Albuquerque to the top one

Reply:
yes:
- "Jaruga_and_Pawlowska_2018" -> chemistry + condensation
- "Kreidenweis_et_al_2003" -> chemistry + condensation
- "Alpert_and_Knopf_2016" -> freezing
- "Ervens_and_Feingold_2012" -> freezing
- "Niedermeier_et_al_2014" -> freezing'